leo/commands/package/add.rs

```diff
@@ -0,0 +1,160 @@
+// Copyright (C) 2019-2021 Aleo Systems Inc.
+// This file is part of the Leo library.
+
+// The Leo library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// The Leo library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
+
+use leo_package::imports::{ImportsDirectory, IMPORTS_DIRECTORY_NAME};
+use tracing::Span;
+
+use std::{
+    fs::{create_dir_all, File},
+    io::{Read, Write},
+};
+
+use crate::{commands::Command, context::Context};
+
+use anyhow::{anyhow, Result};
+use structopt::StructOpt;
+
+use crate::api::Fetch;
```

Comment:
nit: please organize the imports here into two sections - leo specific crates & external crates